src/server_manager/scripts/get_version.mjs

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 The Outline Authors
```

Comment:
It seems this file is not recognized, maybe rebase to the latest `master`?

Reply:
Yeah, I think it will get fixed once I submit fortuna-eslint.